purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1063,10 +1064,12 @@ class Purchases internal constructor(
             val jsonPackagesArray =
                 jsonArrayOfOfferings.getJSONObject(i).getJSONArray("packages")
             for (j in 0 until jsonPackagesArray.length()) {
-                skus.add(
-                    jsonPackagesArray.getJSONObject(j)
-                        .getString("platform_product_identifier")
-                )
+                val sku = jsonPackagesArray.getJSONObject(j)
```

Comment:
@swehner sorry if i missed something -- when would we have platform_product_group_identifier anymore? wasn't that replace with platform_product_identifier?

i know this could just be hacky code, just trying to understand

Reply:
No, we won't anymore - this code was done to support both the old version using group and duration or using the new one using the plan_identifer